### PR TITLE
features(common): added component footer[SEM-06]

### PR DESCRIPTION
**Pull request by sevrinaCh**
Good evening engineer.
I sent you my PR. please check it?

### CODE REVIEW

File: src/components/Footer/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,230 @@
+@import '../../app/variables';
+@import '../../app/mixins';
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@800&display=swap');
```
Comment (Ditmar) [inline src/components/Footer/styles.module.scss:3]: Nice butt. there are a few errors with this approach, ok we know the following, the roboto and Montserrat fonts should be used in the whole system. you need to set up this font globally in you use this approach you only load the font for the footer. and another issue about the use of this approach has to do with performance.
ok the correct way to do this is using the next fonts.
I recommended this article https://nextjs.org/docs/app/building-your-application/optimizing/fonts  please review and adjust your code following the correct approach

File: src/components/Footer/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+        margin-bottom: auto;
+    }
+    
+}
+.imagenFooter{
+   // height: auto;
+    width: 250px;
+    padding-top: 1rem;
+    
+    @include respond-to('small') {
```
Comment (Ditmar) [inline src/components/Footer/styles.module.scss:33]: nice

**Review by Ditmar — CHANGES_REQUESTED** [review]
Nice butt. there are a few errors with this approach, ok we know the following, the roboto and Montserrat fonts should be used in the whole system. you need to set up this font globally in you use this approach you only load the font for the footer. and another issue about the use of this approach has to do with performance.
ok the correct way to do this is using the next fonts.
I recommended this article https://nextjs.org/docs/app/building-your-application/optimizing/fonts  please review and adjust your code following the correct approach

File: src/components/Footer/styles.module.scss (hunk 1 above)
Comment (sevrinaCh) [reply src/components/Footer/styles.module.scss]: Good evening engineer.
I modified what you asked for.
Please check my PR?

**Comment by sevrinaCh** [comment]
Good evening engineer.
I modified what you asked for.
Please check my PR?

**Comment by olmedo90** [comment]
image

It is not according to the figma. verify the correct positioning of the components. 
👇🏼👇🏼👇🏼

image

**Comment by olmedo90** [comment]
image
 It is according to the figma, continue with the same interest. 👍👍👍

**Review by Ditmar — APPROVED** [review]
Approved !

**Pull request merged**